ansible/configs/ibm-rhoic/destroy_env.yml

```diff
@@ -103,7 +103,7 @@
       uri:
         url: "{{ ibm_cloud_api_container_v2_url }}/getCluster?cluster=rhpds-{{ guid }}"
         method: GET
-        status_code: 200,404,500
+        status_code: 200,404,500,502,-1
```

Comment:
200 should be the only code here for success.

Reply:
ack